pcsx2/gui/AppMain.cpp

```diff
@@ -544,7 +544,7 @@ void DoFmvSwitch(bool on)
 				art = AspectRatio_4_3;
 			}
 		}
-		else {
+		else if (!on) {
```

Comment:
I actually wanted you to make the following change:

**old**:
``else``
{
``if ()``
}

**New**:

``else if ()``
{
}

But you did this instead : 
``else if ()``
{
``if ()``
}